tests/mypy/mypy_decorator_t1.py:30: error: Incompatible types in assignment (expression has type "int", variable has type "str")  [assignment]
tests/mypy/mypy_decorator_t1.py:33: error: Too few arguments for "__call__" of "FunctionWrapper"  [call-arg]
tests/mypy/mypy_decorator_t1.py:38: error: Argument 1 to "__call__" of "FunctionWrapper" has incompatible type "str"; expected "int"  [arg-type]
tests/mypy/mypy_decorator_t1.py:41: error: Too many arguments for "__call__" of "FunctionWrapper"  [call-arg]
tests/mypy/mypy_decorator_t1.py:61: error: Too few arguments for "__call__" of "FunctionWrapper"  [call-arg]
tests/mypy/mypy_decorator_t1.py:66: error: Argument 1 to "__call__" of "FunctionWrapper" has incompatible type "str"; expected "int"  [arg-type]
tests/mypy/mypy_decorator_t1.py:69: error: Too many arguments for "__call__" of "FunctionWrapper"  [call-arg]
tests/mypy/mypy_decorator_t1.py:98: error: Incompatible types in assignment (expression has type "str", variable has type "int")  [assignment]
tests/mypy/mypy_decorator_t1.py:101: error: Too few arguments for "function3"  [call-arg]
tests/mypy/mypy_decorator_t1.py:106: error: Argument 1 to "function3" has incompatible type "str"; expected "int"  [arg-type]
tests/mypy/mypy_decorator_t1.py:109: error: Too many arguments for "function3"  [call-arg]
tests/mypy/mypy_decorator_t1.py:134: error: Too few arguments for "__call__" of "FunctionWrapper"  [call-arg]
tests/mypy/mypy_decorator_t1.py:139: error: Argument 1 to "__call__" of "FunctionWrapper" has incompatible type "str"; expected "int"  [arg-type]
tests/mypy/mypy_decorator_t1.py:142: error: Too many arguments for "__call__" of "FunctionWrapper"  [call-arg]
tests/mypy/mypy_decorator_t1.py:217: error: Argument 1 to "__call__" of "FunctionWrapper" has incompatible type "int"; expected "Never"  [arg-type]
tests/mypy/mypy_decorator_t1.py:217: error: Argument 2 to "__call__" of "FunctionWrapper" has incompatible type "int"; expected "Never"  [arg-type]
tests/mypy/mypy_decorator_t1.py:219: error: Argument 1 to "__call__" of "FunctionWrapper" has incompatible type "int"; expected "Never"  [arg-type]
tests/mypy/mypy_decorator_t1.py:220: error: Argument 1 to "__call__" of "FunctionWrapper" has incompatible type "int"; expected "Never"  [arg-type]
tests/mypy/mypy_decorator_t1.py:220: error: Argument "b" to "__call__" of "FunctionWrapper" has incompatible type "int"; expected "Never"  [arg-type]
tests/mypy/mypy_decorator_t1.py:228: error: Argument 1 to "__call__" of "FunctionWrapper" has incompatible type "int"; expected "Never"  [arg-type]
tests/mypy/mypy_decorator_t1.py:228: error: Argument 2 to "__call__" of "FunctionWrapper" has incompatible type "int"; expected "Never"  [arg-type]
tests/mypy/mypy_decorator_t1.py:230: error: Argument 1 to "__call__" of "FunctionWrapper" has incompatible type "int"; expected "Never"  [arg-type]
tests/mypy/mypy_decorator_t1.py:231: error: Argument 1 to "__call__" of "FunctionWrapper" has incompatible type "int"; expected "Never"  [arg-type]
tests/mypy/mypy_decorator_t1.py:231: error: Argument "b" to "__call__" of "FunctionWrapper" has incompatible type "int"; expected "Never"  [arg-type]
tests/mypy/mypy_decorator_t1.py:239: error: Argument 1 to "__call__" of "FunctionWrapper" has incompatible type "int"; expected "Never"  [arg-type]
tests/mypy/mypy_decorator_t1.py:239: error: Argument 2 to "__call__" of "FunctionWrapper" has incompatible type "int"; expected "Never"  [arg-type]
tests/mypy/mypy_decorator_t1.py:241: error: Argument 1 to "__call__" of "FunctionWrapper" has incompatible type "int"; expected "Never"  [arg-type]
tests/mypy/mypy_decorator_t1.py:242: error: Argument 1 to "__call__" of "FunctionWrapper" has incompatible type "int"; expected "Never"  [arg-type]
tests/mypy/mypy_decorator_t1.py:242: error: Argument "b" to "__call__" of "FunctionWrapper" has incompatible type "int"; expected "Never"  [arg-type]
Found 29 errors in 1 file (checked 1 source file)
